cmd/parser/parser.go

```diff
@@ -35,6 +35,22 @@ func CreateParser(librariesPath []string) *Parser {
 	return parser
 }
 
+func (p *Parser) loadFile(filePath string) (map[string]interface{}, error) {
+	fileExt := filepath.Ext(filePath)
+	switch fileExt {
+	case ".jsonnet":
+		return p.loadJsonnetFile(filePath)
+	case ".json":
+		return p.loadJSONFile(filePath)
+	case ".yaml":
+		return p.loadYAMLFile(filePath)
+	case ".yml":
```

Comment:
Can you merge these cases?

Reply:
I forgot I could make a case list :P thanks.